### PR TITLE
Update BLE examples.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/ble/scanning.mdx [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # Scanning
 
-To start scanning and for nearby BLE devices, simply call `scan` on the `ble.Device`. The provided block will be called for each scan result:
+To start scanning and for nearby BLE devices, simply call `scan` on the adapter's
```
Comment (erikcorry) [inline docs/tutorials/ble/scanning.mdx:3]:
```suggestion
To start scanning for nearby BLE devices, simply call `scan` on the adapter's
```

**Review by erikcorry — APPROVED** [review]

File: docs/tutorials/ble/client.mdx [hunk 2]
```diff
@@ -9,27 +9,27 @@ In order for a BLE device to establish a connection to a remote device, the devi
...
+SCAN_DURATION ::= Duration --s=3
+
+find_with_service central/ble.Central service/ble.BleUuid:
+  central.scan --duration=SCAN_DURATION: | device/ble.RemoteScannedDevice |
+    if device.data.service_classes.contains service:
+        return device.address
   throw "no device found"
 ```
 
 With the address available, the connection can be established. In the following example, we use the `find_with_service` function to find a remote device with a battery service and establish a connection to that device.
```
Comment (mikkeldamsgaard) [inline docs/tutorials/ble/client.mdx:22]: With the address available, the connection can be established. In the following example, we use the `find_with_service` function to find a remote device with an advertised battery service and establish a connection to that device. Note that the device might also support services that are not explicitly advertised,

File: docs/tutorials/ble/scanning.mdx [hunk 3]
```diff
@@ -24,17 +26,18 @@ The following example shows how to scan for 3 seconds for the addresses of remot
 ```toit
 import ble
 
-BATTERY_SERVICE ::= ble.uuid 0x180F
+BATTERY_SERVICE ::= ble.BleUuid "180F"
```
Comment (mikkeldamsgaard) [inline docs/tutorials/ble/scanning.mdx:29]: Again, above this: e.g. implement a specific service should be e.g. advertising a specific service. 
And in the text, if you create your own full 128-bit UUID you can add your custom service.

File: docs/tutorials/ble/client.mdx (hunk 2 above)
Comment (floitsch) [reply docs/tutorials/ble/client.mdx]: done.

File: docs/tutorials/ble/scanning.mdx (hunk 3 above)
Comment (floitsch) [reply docs/tutorials/ble/scanning.mdx:29]: done.

**Pull request merged**